src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const sourceStringArr = sourceString.split(';');
+
+  for (const rule of sourceStringArr) {
```

Comment:
CL-5 [CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead.
you can use for example the .map method